autogpt_platform/backend/backend/copilot/codex_model_router_test.py

```
@@ -0,0 +1,197 @@
+from unittest.mock import AsyncMock
+
+import pytest
+
+import backend.data.llm_registry.registry as registry
```

Ruff (0.16.1)

[warning] Use `from backend.data.llm_registry import registry` in lieu of alias

Replace with `from backend.data.llm_registry import registry`

(PLR0402)